spec/support/request/web_helper.rb

```diff
@@ -95,16 +95,29 @@ def wait_until(secs = nil)
     end
   end
 
+  def within_row(num, &block)
+    within("table.index tbody tr:nth-child(#{num})", &block)
+  end
+
   def select2_select(value, options)
     id = options[:from]
     options[:from] = "#s2id_#{id}"
     targetted_select2(value, options)
   end
 
+  def targetted_select2(value, options)
+    # find select2 element and click it
+    find(options[:from]).find('a').click
+    select_select2_result(value)
+  end
+
+  def select_select2_result(value)
+    sleep(1)
```

Comment:
:see_no_evil: can you try removing this and use `have_content` or similar if needed?

Reply:
I am just bringing this from Spree in this PR.
I tried removing this but it breaks the specs.
have_content is not appropriate, this method is actually selecting the select2 result with mouseup.
It's dodgy but it's one of those things we need first to bring to our side, and fix in a second step. There will be a few like these in the bye bye spree PRs.

Reply:
could you create an issue for it so we don't lose track?

Reply:
yes, done: https://github.com/openfoodfoundation/openfoodnetwork/issues/5972
this could save us a few minutes of build time :+1:

Reply:
you said MINUTES :scream:

Reply:
yes, it's called dozens of times in the specs :+1:  I moved it to the top of "tech debt prioritized" now :-)